src/Perlang.Tests.Integration/Number/NumberTests.cs

```diff
@@ -95,9 +97,12 @@ public void literal_negative_integer()
             Assert.Equal(-123, result);
         }
 
-        [Fact]
-        public void literal_float()
+        [Theory]
+        [ClassData(typeof(TestCultures))]
+        public async Task literal_float(CultureInfo cultureInfo)
         {
+            CultureInfo.CurrentCulture = cultureInfo;
+
```

Comment:
An important aspect regarding this is that changing the `CurrentCulture` _without making the test method be `async Task`_ would be dangerous, since the culture change would leak over to other test methods as well. Given this, I think we should add an ArchUnit.NET test to ensure that all test methods which take a `CultureInfo` parameter **must** be `async` and return a `Task`.

(The latter check is in fact not mandatory to check, since there's already an inspection in place that will cause errors on `async void` methods, which is definitely a dangerous anti-pattern that should be avoided.)